getting-started/kubearmor_visibility.md

```diff
@@ -0,0 +1,128 @@
+# KubeArmor Visibility
+
+KubeArmor currently supports enabling visibility for containers/hosts.
+
+Visibility for containers is enabled by default and can be accessed using the `karmor` tool.
+
+
+<details>
+  <summary>Know More On Visibillity</summary>
+
+#### KubeArmor provides visibility on the following behavior of containers
+* Process
+* Files
+* Networks
+
+</details>
+
+
+## Getting Container Visibility
+
+##### Prerequisites
+
+* If you don't have access to a K8s cluster, please follow  [this](https://github.com/kubearmor/KubeArmor/blob/main/getting-started/deployment_guide.md#prerequisites) to set one up.
+* To install karmor CLI tool in the cluster follow [here](https://github.com/kubearmor/KubeArmor/blob/main/getting-started/deployment_guide.md#1-download-and-install-karmor-cli-tool)
+
+#### Example: wordpress-mysql
+
+* To deploy [wordpress-mysql](https://github.com/kubearmor/KubeArmor/blob/main/examples/multiubuntu.md) app follow [this](https://github.com/kubearmor/KubeArmor/blob/main//examples/wordpress-mysql.md)
+* Now we need to deploy some sample policies
+```
+kubectl apply -f https://raw.githubusercontent.com/kubearmor/KubeArmor/main/examples/wordpress-mysql/security-policies/ksp-wordpress-block-process.yaml
+```
+This sample policy blocks execution of the `apt` and `apt-get` commands in wordpress pods with label selector `app: wordpress`.
+
+* Checking default visibility
+
+  * Container visibility is enabled by default, in this case, we can check it using `kubectl describe` and greping `kubearmor-visibility `
+
+  ```text
+  $ POD_NAME=$(kubectl get pods -n wordpress-mysql -l app=wordpress -o jsonpath='{.items[0].metadata.name}') && kubectl describe -n wordpress-mysql pod $POD_NAME | grep kubearmor-visibility
```

Comment:
Can we remove the $ sign so that the user can directly copy-paste the entire command?